processes/hathiTrust.py

```diff
@@ -10,6 +10,14 @@
 from mappings.hathitrust import HathiMapping
 from logger import createLog
 
+import newrelic.agent
+
+if os.environ.get('NEW_RELIC_LICENSE_KEY', None):
```

Comment:
I _think_ that this is necessary because the `newrelic.agent` needs to be told about it's configuration the first time we create a `background_task`.

The idea that I have is that if we instrument a background task in `main.py` (see my comment there). That we could delete this and instrument these methods _within_ that task and they would get captured.